stock_picking_delivery_link/models/stock_picking.py

```diff
@@ -55,7 +55,7 @@ def _pre_put_in_pack_hook(self, move_line_ids):
         else:
             return res
 
-    def _set_delivery_package_type(self):
+    def _set_delivery_package_type(self, batch_pack=False):
         self.ensure_one()
         res = super()._set_delivery_package_type()
```

Comment:
Hi anh @nilshamerlinck, it does not work. 

- As I ran test, it failed because of this error:

> TypeError: StockPicking._set_delivery_package_type() takes 1 positional argument but 2 were given

Reply:
is your odoo codebase up-to-date? 
this params was actually added 2 months ago upstream, which explains why we face the issue now only: https://github.com/odoo/odoo/commit/4d9af42f04aa9c0fbc4924cae3abb9f2d83db66c
you can add this reference in the final oca PR